src/Common/ORDataObject/ORDataObject.php

```diff
@@ -102,7 +102,7 @@ protected function populate_array($results)
      * @param boolean $blank optional value to include a empty element at position 0, default is true
      * @return array array of values as name to index pairs found in the db enumeration of this field
      */
-    protected function _load_enum($field_name, $blank = true)
+    public function _load_enum($field_name, $blank = true)
```

Comment:
hi @yashrajbothra , you are ok with protected on the above 4 functions (always better to keep private/protected if possible). Only need public for the _utility_array function.

Reply:
Actually I saw another class using one more function. So did it with all.
I think except those two others can be protected.